NOTICE

```diff
@@ -3,3 +3,7 @@ Copyright [2014-2016] The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+Portions of this software were developed at
```

Comment:
I just pinged someone who's better at this than I am.  I wouldn't use Mynewt as an example as they are incubating, but you are correct (and I even +1'd their release that included this).  I will point the link you're quoting includes this:

```
- remove such notices, or
- move them to the NOTICE file associated with each applicable project release, or
- provide written permission for the ASF to make such removal or relocation of the notices.
```

Which the SGA should fall under the 1st or 3rd bullet point